spark/unitycatalog/src/test/scala/com/sparkuctest/UnityCatalogSupport.scala

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.sparkuctest
```

Comment:
Is `org.apache.spark.sql.delta.test.unty` a better naming?

Reply:
btw,   should we write the test in scala or java ?  will it be better to use Java to maintain those tests ?